src/manifolds/MetricManifold.jl

```diff
@@ -62,9 +206,9 @@ abstract type RiemannianMetric <: AbstractMetric end
 Compute the Christoffel symbols of the first kind in local coordinates of basis `B`.
 The Christoffel symbols are (in Einstein summation convention)
 
-```math
+````math
```

Comment:
Is it supposed to have three or four backticks? We are inconsistent about it.

Reply:
From Markdown both are fine; I prefer (and use most often) 3, I do not know why exactly. But overall our docs are not consistent in that, that's right.

Reply:
OK.

Reply:
I also think we do not have to unify that, Markdown says any number of backpacks is fine and I would mostly use 3 but if I have. code block in my code block I would use 4 and for the inner one 3 :)